design/task_model.md

```diff
@@ -127,8 +127,8 @@ that the requested node has the necessary resources, in the case of global
 services' tasks). It changes their state to `ASSIGNED`.
 
 From this point, control over the state passes to the agent. A task will
-progress through the `ACCEPTED`, `PREPARING`, and `STARTING` states on the
-way to `RUNNING`. If a task exits without an error code, it moves to the
+progress through the `ACCEPTED`, `PREPARING`, `READY', and `STARTING` states on
+the way to `RUNNING`. If a task exits without an error code, it moves to the
```

Comment:
>If a task exits without an error code, it moves to the `COMPLETE` state.

Lets also clarify this by saying that the actual state is changed to COMPLETE by the agent.

Reply:
Actually, it says the agent handles these states on the line above the bit shown by GitHub's diff, so maybe it's OK as it is.